RasimKhusaenov/my_own_blog
# Fix footer layout

### Summary

- Fixed footer
- Separated scss styles
- Fixed navbar padding
- Fixed wrong width of page

### How it works

![image](https://user-images.githubusercontent.com/35406008/174610044-614b9e29-a258-488e-9a61-84a225848eb7.png)

### Test plan

List of steps to manually test introduced functionality:

- Go to https://my-own-blog-pr-61.herokuapp.com/
- Make sure that footer is displayed correctly

### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.

### Checklist:


### References
- Resolves #59 Issue
